manticore/config.py

```diff
@@ -0,0 +1,19 @@
+import configparser
+import os
+
+
+def _config(path=None) -> configparser.ConfigParser:
```

Comment:
Mini docstring documenting the precedence?
This will read the the configuration from a .ini file in `path` or else the `./.manticorerc` or else `~/.manticorerc`